eftest/src/eftest/runner.clj

```diff
@@ -18,15 +19,23 @@
 (defn- test-vars [ns vars opts]
   (let [once-fixtures (-> ns meta ::test/once-fixtures test/join-fixtures)
         each-fixtures (-> ns meta ::test/each-fixtures test/join-fixtures)
+        capture-context (capture/init-capture)
         report        (synchronize test/report)
-        test-var      (fn [v] (binding [test/report report] (test/test-var v)))]
+        test-var      (fn [v]
+                        (capture/begin-capture)
+                        (binding [test/report report
+                                  test/*test-out* *out*
```

Comment:
Doesn't this override any custom `*test-out*` settings?

Reply:
I'm not sure about that.

Reply:
Could you briefly explain the purpose of this line?